### DISCUSSION TITLE
Feed need some adjustments and improvement

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#63201] [Other Feature Feedback, Questions, & Ideas]
Well, the feed was made so we could find repositories which we want to, like I made so many discord bots, so GitHub takes data from what kind of threads I am joined in, what kind of repositories I make and what I title them, what tags I add in them and also what kind of other repositories have similar title, tags and code files in them, GitHub takes all of it and gives us the feed

But to be honest, this feed system is always boring, it doesn't have variety like YouTube.
Like I cannot set filter to "today's trending/most starred", "this week's most starred", "this past month's most starred", "this past year's most starred", and they do not even have a search bar so I could search in feeds page.

Edit: Oh well damn it now it see it, you people have a different page name "trending repositories", I am the most confused person ever on this earth

**Comment by XessiveObserver** [comment]
Then its over rushed IMO
